### PR TITLE
adding testing for ubuntu 16.04 and proxy config for HPE network

### PR DESCRIPTION
**Pull request by majormoses**

### CODE REVIEW

**Comment by jsmartt** [comment]
retest this please

**Comment by jsmartt** [comment]
retest this please

**Comment by jsmartt** [comment]
@majormoses , looks like this is converging successfully after all

File: .kitchen.docker.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 driver:
   name: docker
   use_sudo: false
+  http_proxy: <%= ENV['http_proxy'] %>
```
Comment (huynt1979) [inline .kitchen.docker.yml]: What are these?

Comment (jsmartt) [inline .kitchen.docker.yml]: These are necessary to make docker build and run inside the HPE network, but when specified this way, it should only affect people that have the http_proxy environment variable set

Comment (huynt1979) [inline .kitchen.docker.yml]: I would suggest 2 things:
- Put a comment to make clear the purpose of these settings.
- You can embed Ruby code within yaml file to make its content even more dynamic. See this: https://github.com/CloudCruiser/ops_chef-cc_service/blob/master/.kitchen.dokken.yml#L42-L49

**Comment by jsmartt** [comment]
OK, let me know what you think of it now @huynt1979

**Pull request merged**